src/config/routes/routes.tsx

```diff
@@ -542,10 +542,28 @@ export const platformRoutes: RouteObject[] = [
         path: getPath('discoveryActivity'),
         async lazy() {
           const Activity = await Discovery().then((m) => m.Activity)
-          return {
-            element: renderPrivateRoute(Activity),
-          }
+          return { Component: Activity }
         },
+        children: [
+          {
+            index: true,
+            element: <Navigate to={getPath('discoveryActivityGlobal')} replace />,
+          },
```

Comment:
wouldn't this always send user's to the global feed, I think if the user is logged in we want them to be sent to, followed projects, if not logged in then to the global feed.

You can create a component and add the <Navigate> based on the login status.

Reply:
For more context, we decided to keep both routes private thus requiring users to login on both `global` and `projectsIFollow` feeds. 

To answer your question, no it doesn't always send the user to the `global feed` that been said I updated it to `projects I follow` to fit with the default in the component.